### DISCUSSION TITLE
Support logging contents of wp-content/debug.log

### DISCUSSION THREAD

**Discussion by sebastiandedeyne** [spatie/wordpress-ray#70] [Ideas]
Feature idea: https://twitter.com/tailcode_studio/status/1687197926668521472

**Comment by jeffreyvr** [comment]
While it is not exactly the same as the feature suggested, occurrences of `WP_Error` can be logged with:

```php
ray()->showWordPressErrors();
```

The `WP_Error` object is used in a variety of ways, including on fatal errors.

<img width="856" alt="Screenshot 2023-08-04 at 14 51 01" src="https://github.com/spatie/wordpress-ray/assets/9550079/37ac2e5c-5412-4503-92d8-4ffed54520de">

You probably already knew this, but I thought I'd mention it just in case.

That being said, it would be nice to have something that only logs the errors that would only end up in `debug.log`.
